Makefile.in

```diff
@@ -195,11 +195,16 @@ ifeq ($(shell uname), Darwin)
 endif
 
 #
-# All translation message catalogs
+# All translation message catalogs, filter files based on LINGUAS.
 #
 TRANSLATIONS_SRC := $(wildcard po/*.po)
-ifdef HAVE_GETTEXT
+ifeq ($(HAVE_GETTEXT), 1)
```

Comment:
On my system (macOS Sierra) the *Makefile* ends up with this line: `HAVE_GETTEXT=`. So if that results in `ifdef HAVE_GETTEXT`evaluating true, thus necessitating this change, then the same change needs to occur at line 767. Otherwise this change seems to be unnecessary. Or have I had one too many beers this evening?